src/main/java/seedu/address/logic/parser/ParserUtil.java

```diff
@@ -50,6 +55,59 @@ public static Name parseName(String name) throws ParseException {
         return new Name(trimmedName);
     }
 
+    /**
+     * Parses a {@code String name} into a {@code Name}.
+     * Leading and trailing whitespaces will be trimmed.
+     *
+     * @param clientId in string format from user input
+     * @return Client of the client id
+     * @throws ParseException if the fiven {@code client} is invalid.
+     */
+    public static Client parseClient(String clientId) throws ParseException {
+        requireNonNull(clientId);
+        String trimmedClientId = clientId.trim();
+        ClientId clientIdRes = new ClientId(Integer.parseInt(trimmedClientId));
```

Comment:
Perhaps `NumberFormatException` could be caught here

Reply:
Also needs to handle the case where the `ClientId` is not empty, as the project created should be added to the `Client`'s list of `Projects`